package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@digitalbazaar/ecdsa-multikey",
+  "version": "1.0.0",
+  "description": "Javascript library for generating and working with EcdsaMultikey key pairs.",
+  "homepage": "https://github.com/digitalbazaar/ecdsa-multikey",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/ecdsa-multikey"
+  },
+  "license": "BSD-3-Clause",
+  "type": "module",
+  "exports": "./lib/index.js",
+  "browser": {
+    "./lib/ecdsa.js": "./lib/ecdsa-browser.js"
+  },
+  "files": [
+    "lib/**/*.js"
+  ],
+  "dependencies": {
+    "base58-universal": "^2.0.0",
+    "base64url-universal": "^2.0.0",
+    "multiformats": "^11.0.1"
```

Comment:
Used?

Reply:
At one point I was using it, but not anymore. I have removed it for now, but we may need it in the future, since `multicodec` is deprecated and will not support future additions to the `multiformats/multicodec` table.